src/utils.js

```diff
@@ -1288,3 +1288,16 @@ export function cyrb53Hash(str, seed = 0) {
   h2 = imul(h2 ^ (h2 >>> 16), 2246822507) ^ imul(h1 ^ (h1 >>> 13), 3266489909);
   return (4294967296 * (2097151 & h2) + (h1 >>> 0)).toString();
 }
+
+export function removeDuplicatesFromObjectArray(array, key) {
```

Comment:
unit test cases needed

Reply:
can we avoid this function with arrayName.filter(uniques)?
Refer: https://github.com/prebid/Prebid.js/blob/master/src/utils.js#L612

Reply:
this array is an arry of objects of the format {id: '1'}. so a normal array filtering function wont work.

Reply:
@pm-manasi-moghe  we can use the filter function as below ...
```
var keyName = 'id'; 
var arr = [ {id: 1}, {id: 10}, {id: 1}, {id: 2}, {id: 3}, {id: 10}, {id:4}];
arr.filter(function(filterValue, filterIndex, arry){
    return arry.slice(filterIndex+1).find(function(findValue, findIndex){
        return findValue[keyName] == filterValue[keyName] 
    }) == undefined
});
```

Reply:
made the required changes

Reply:
@pm-manasi-moghe we will need to add the unit test cases for this utils function